src/frontend/src/lib/styles/global/gix.scss

```diff
@@ -60,7 +60,7 @@
 
 	--alert-width: calc(100% - var(--padding-8x));
 	--alert-max-width: calc(100vw - var(--padding-4x));
-	--alert-max-height: calc(100vh - var(--padding-4x));
+	--alert-max-height: calc(100dvh - var(--padding-4x));
```

Comment:
`dvh` is not supported yet on Firefox (https://caniuse.com/?search=dvh) we should use the same snippet as @AntonioVentilii-DFINITY introduced last Friday.

Reply:
@peterpeterparker isn't it just the static method `dvh()` that is not supported by Firefox? The Viewport units that i used are supported.
<img width="1067" alt="image" src="https://github.com/user-attachments/assets/50313762-6339-4ccb-9841-f932d8e8af3d">

Reply:
Oh great then, my bad!

cc @AntonioVentilii-DFINITY